.github/workflows/test.yml

```diff
@@ -14,88 +14,55 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Build with Mono
+    - name: Test with NUnit on Mono
       run: |
         chmod +x build.sh
-        $GITHUB_WORKSPACE//build.sh -v
-    - name: Test with Nunit on Mono
-      run: $GITHUB_WORKSPACE//test.sh integration
+        $GITHUB_WORKSPACE//build.sh --verbosity=diagnostic --target=test --testExecutionType=all
```

Comment:
The `TESTING.md` file should also be updated with the new way of running the integration tests.

Reply:
Ah, good shout! I will get that updated!